docs/config.md

```diff
@@ -145,7 +145,8 @@ environment variables for the OTLP exporter:
 
 > ⚠️ **Do NOT use in production.**
 >
-> Prometheus exporter uses [an unreliable, unmaintained legacy .NET component](https://github.com/dotnet/runtime/issues/28658#issuecomment-462062760).
+> Prometheus exporter [is intended for dev inner-loop,
+> there is no plan to make it production ready](https://github.com/open-telemetry/opentelemetry-dotnet/blob/622583c6ffaea2e3d83d4fcb8935c6a8dbc6f81f/src/OpenTelemetry.Exporter.Prometheus.HttpListener/README.md?plain=1#L10-L16).
```

Comment:
Should we call out that this is the "HttpListener" version? 

After reading [this](https://github.com/open-telemetry/opentelemetry-dotnet/blob/622583c6ffaea2e3d83d4fcb8935c6a8dbc6f81f/src/OpenTelemetry.Exporter.Prometheus.HttpListener/README.md?plain=1#L10-L16) the user may ask how to use `OpenTelemetry.Exporter.Prometheus.AspNetCore`?

Personally, I think it would be easier if we simply:

```suggestion
> Prometheus exporter is intended for the inner dev loop.
> Production environments can use a combination of OTLP exporter
> with [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector-releases)
> having [`otlp` receiver](https://github.com/open-telemetry/opentelemetry-collector/tree/v0.61.0/receiver/otlpreceiver) and [`prometheus` exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.61.0/exporter/prometheusexporter).
```

I believe that we might add support for the AspNetCore version of the Prometheus exporter in the future.